### ISSUE TITLE
Problem summing losses for VAE

### ISSUE THREAD

**Issue by callum-b** [keras-team/keras-io#983]
The VAE class in the example for VAE shows `total_loss = reconstruction_loss + kl_loss`, but this is not the case when adding these values in the `fit` output. I ran the source code from the example and ran it adapted to my own data, same mistake. How come? Should the loss calculations be adjusted during training?
![vae_losses](https://user-images.githubusercontent.com/39163447/178772255-a5700f65-22f7-46da-bcdd-ef13495fe044.PNG)
